### PR TITLE
Added Pokémon Ultra Sun and Ultra Moon.

### PR DESCRIPTION
**Pull request by Hexagon12**

### CODE REVIEW

File: games/pokemon-ultra-sun-ultra-moon/game.dat [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title = "Pokémon Ultra Sun and Ultra Moon"
+description = "Pokémon Ultra Sun and Pokémon Ultra Moon are role-playing video games developed by Game Freak, published by The Pokémon Company, and distributed by Nintendo for the Nintendo 3DS. Announced in June 2017, the titles are enhanced versions of Pokémon Sun and Moon with a modified storyline, similar to previous upper versions like Platinum, Emerald and Yellow. The games are the second installments in the seventh generation of the main series and the last planned installment of the main series Pokémon games for Nintendo 3DS."
```
Comment (BreadFish64) [inline games/pokemon-ultra-sun-ultra-moon/game.dat]: > upper versions

sounds a little awkward

File: games/pokemon-ultra-sun-ultra-moon/game.dat [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+needs_system_files = true
+needs_shared_font = false
+
+[[ releases ]]
+title = "00040000001b5000"
+region = "all"
+release_date = "2017-11-17"
+
+[[ releases ]]
+title = "00040000001b5100"
```
Comment (TheKoopaKingdom) [inline games/pokemon-ultra-sun-ultra-moon/game.dat]: Ditto.

File: games/pokemon-ultra-sun-ultra-moon/game.dat [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+region = "all"
+release_date = "2017-11-17"
+
+[[ testcases ]]
+title = "00040000001b5100"
```
Comment (TheKoopaKingdom) [inline games/pokemon-ultra-sun-ultra-moon/game.dat]: Ditto.

**Comment by Leo626** [comment]
LGTM but those screenshots look like they have the no outline hack applied?

**Comment by Hexagon12** [comment]
I didn't use any hacks for taking the screenshots.

**Pull request merged**